src/main/java/at/hannibal2/skyhanni/config/features/chat/ChatConfig.java

```diff
@@ -84,22 +85,22 @@ public String toString() {
     public CompactPotionConfig compactPotionMessages = new CompactPotionConfig();
 
     @Expose
-    @ConfigOption(name = "Compact Bestiary Message", desc = "Shorten the Bestiary level up message, showing additional information when hovering.")
+    @ConfigOption(name = "Compact Bestiary Messages", desc = "Compact the Bestiary level up message, only showing additional information when hovering.")
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean compactBestiaryMessage = true;
 
     @Expose
-    @ConfigOption(name = "Arachne Hider", desc = "Hide chat messages about the Arachne Fight while outside of §eArachne's Sanctuary§7.")
+    @ConfigOption(name = "Arachne Fight Messages", desc = "Hide chat messages about the Arachne Fight while outside of §eArachne's Sanctuary§7.")
```

Comment:
please keep the word "hider" over "messsages". this change is less descriptive and just longer (smaller formatted in game).